cluster-up/cluster/kind-1.27/provider.sh

```diff
@@ -14,8 +14,8 @@ else
 fi
 
 function set_kind_params() {
-    export KIND_VERSION="${KIND_VERSION:-0.19.0}"
-    export KIND_NODE_IMAGE="${KIND_NODE_IMAGE:-kindest/node:v1.27.1@sha256:b7d12ed662b873bd8510879c1846e87c7e676a79fefc93e17b2a52989d3ff42b}"
+    export KIND_VERSION="${KIND_VERSION:-0.20.0}"
+    export KIND_NODE_IMAGE="${KIND_NODE_IMAGE:-kindest/node:v1.27.3@sha256:3966ac761ae0136263ffdb6cfd4db23ef8a83cba8a463690e98317add2c9ba72}"
```

Comment:
wdyt about updating node and folder name ?
thanks

Reply:
> wdyt about updating node to 1.28 and folder name ? (reason is that we already use 1.28 for the vm based providers) thanks

Ok, I can give it a try.

Reply:
Thanks, btw no rush of course, can be done on a follow PR if you prefer

Reply:
> Thanks, btw no rush of course, can be done on a follow PR if you prefer

That would be better, as it block the e2e tests on Arm64 platform.